scripts/init.sql

```diff
@@ -1,158 +1,146 @@
-    );
-
-CREATE TABLE
-    IF NOT EXISTS messages(
-        id integer NOT NULL,
-        "content" varchar(500) NOT NULL,
-        sender integer NOT NULL,
-        wishcard_id integer NOT NULL,
-        created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
-        CONSTRAINT messages_pkey PRIMARY KEY(id)
-    );
-
-CREATE TABLE
-    IF NOT EXISTS orders(
-        id integer NOT NULL,
-        status integer NOT NULL DEFAULT 0,
-        delivery_date timestamp,
-        tracking_info varchar(500),
-        donor_id integer NOT NULL,
-        child_id integer NOT NULL,
-        item_id integer NOT NULL,
-        agency_id integer NOT NULL,
-        created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
-        updated_at timestamp,
-        CONSTRAINT orders_pkey PRIMARY KEY(id)
-    );
-
-CREATE TABLE
-    IF NOT EXISTS users(
-        id integer NOT NULL,
-        first_name varchar(255) NOT NULL,
-        last_name varchar(255) NOT NULL,
-        email varchar(255) NOT NULL,
-        "password" varchar(100) NOT NULL,
-        "role" integer NOT NULL,
-        login_mode integer NOT NULL,
-        bio varchar(500),
-        verified boolean NOT NULL DEFAULT false,
-        image_id integer NOT NULL,
-        created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
-        updated_at timestamp,
-        deleted_at timestamp,
-        CONSTRAINT users_pkey PRIMARY KEY(id)
-    );
-
-CREATE TABLE
-    IF NOT EXISTS verification_tokens(
-        id integer NOT NULL,
-        token varchar(255) NOT NULL,
-        "type" integer NOT NULL,
-        expiration timestamp NOT NULL,
-        user_id integer NOT NULL,
-        created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
-        CONSTRAINT verification_tokens_pkey PRIMARY KEY(id)
-    );
-
-CREATE TABLE
-    IF NOT EXISTS verifications(
-        id integer NOT NULL,
-        email_verified boolean NOT NULL,
-        user_id integer NOT NULL,
-        created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
-        updated_at timestamp,
-        CONSTRAINT verifications_pkey PRIMARY KEY(id)
-    );
-
-CREATE TABLE
-    IF NOT EXISTS wishcards(
-        id integer NOT NULL,
-        address_line_1 varchar(255) NOT NULL,
-        address_line_2 varchar(255),
-        city varchar(100) NOT NULL,
-        state varchar(100) NOT NULL,
-        country varchar(100) NOT NULL,
-        zip_code varchar(50) NOT NULL,
-        status integer NOT NULL DEFAULT 0,
-        created_by integer NOT NULL,
-        agency_id integer NOT NULL,
-        child_id integer NOT NULL,
-        item_id integer NOT NULL,
-        image_id integer NOT NULL,
-        order_id integer NOT NULL,
-        created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
-        updated_at timestamp,
-        CONSTRAINT wishcards_pkey PRIMARY KEY(id)
-    );
-
--- constraints
+SET check_function_bodies = false ;
+
+CREATE TABLE agencies(
+    id uuid NOT NULL DEFAULT gen_random_uuid(),
```

Comment:
```suggestion
    agency_id uuid NOT NULL DEFAULT gen_random_uuid(),
```

@Enubia What are your thoughts on having table-specific ID columns like this so we aren't consistently dealing with the ambiguous column names error when doing joins?

Reply:
I'm used to the primary key being called `id`.
Personally, I like `<variable>.id` more.
If I, for example, need to access the user id, it would be a bit redundant to call `user.user_id` in my opinion.
But let's see what @stacysealky thinks. I'm open to changes of course.

Reply:
I'm with @Enubia on this one. I'm in favor of just calling it id to avoid redundancy. 
We can use an Alias for join to avoid the ambiguous column name error, right? @nick-w-nick

Reply:
@stacysealky That is correct, this was just an attempt to prevent needing to do that, as it is against best practice to name them generically instead of by table, but I don't mind doing it this way instead.